index.bs

```diff
@@ -2645,12 +2646,14 @@ WebAuthn supports multiple attestation types:
     any specific attestation key. Instead it uses the credential private key to create the attestation signature.
     Authenticators without meaningful protection measures for an attestation private key typically use this attestation type.
 
-: <dfn>Privacy CA</dfn>
-:: In this case, the Authenticator owns an authenticator-specific (endorsement) key. This key is used to securely communicate
-    with a trusted third party, the Privacy CA.  The Authenticator can generate multiple attestation key pairs and asks the
-    Privacy CA to issue an attestation certificate for it. Using this approach, the Authenticator can limit the exposure of the
-    endorsement key (which is a global correlation handle) to Privacy CA(s). Attestation keys can be requested for each [=public
-    key credential=] individually.
+: <dfn>Attestation CA</dfn>
+:: In this case, an [=authenticator=] is based on a Trusted Platform Module (TPM) and holds an authenticator-specific
+    "endorsement key" (EK). This key is used to securely communicate with a trusted third party, the [=Attestation CA=]
+    [[!TCG-CMCProfile-AIKCertEnroll]] (formerly known as a "Privacy CA"). The [=authenticator=] can generate multiple
+    attestation identity key pairs (AIK) and requests an [=Attestation CA=] to issue an attestation identity certificate (AIK)
+    for each. Using this approach, such an [=authenticator=] can limit the exposure of the EK (which is a global correlation
+    handle) to Attestation CA(s). AIKs can be requested for each [=authenticator=]-generated [=public key credential=]
```

Comment:
Ditto

Reply:
Ditto ;-)

Reply:
I am thinking usage of AIK here is nominally correct.

Reply:
Yeah, this looks better!